common/models/user.json

```diff
@@ -76,6 +76,12 @@
       "permission": "ALLOW",
       "property": "resetPassword",
       "accessType": "EXECUTE"
+    },
+    {
+      "principalType": "ROLE",
+      "principalId": "$owner",
+      "permission": "ALLOW",
+      "property": "replaceById"
```

Comment:
As mentioned elsewhere, please move this new ACL entry closer to the entry for `updateAttributes`.